src/GovUk.Education.ExploreEducationStatistics.Common/Utils/ComparerUtils.cs

```diff
@@ -39,5 +40,10 @@ public int GetHashCode(T obj)
                 return _propertyGetter.Invoke(obj).GetHashCode();
             }
         }
+
+        public static bool SequencesAreEqualIgnoringOrder<T>(IEnumerable<T> left, IEnumerable<T> right)
+        {
+            return left.OrderBy(id => id).SequenceEqual(right.OrderBy(id => id));
+        }
     }
```

Comment:
This is going to raise an exception if  type `T` can't be compared in `OrderBy`.

I would add `where T : IComparable` here to make sure the elements can be compared.

That should be fine if it's always being used with `List<Guid>` or `List<int>` etc.

You might need to add `where T : IComparable` in a few of the other methods which are using this if they are generic too.